lisa/sut_orchestrator/azure/features.py

```diff
@@ -515,25 +515,27 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self._initialize_information(self._node)
         self._is_nvidia = True
 
-    @classmethod
-    def _install_by_platform(cls, *args: Any, **kwargs: Any) -> None:
-        template: Any = kwargs.get("template")
-        environment = cast(Environment, kwargs.get("environment"))
-        log = cast(Logger, kwargs.get("log"))
-        log.debug("updating arm template to support GPU extension.")
-        resources = template["resources"]
-
-        # load a copy to avoid side effect.
-        gpu_template = json.loads(cls._gpu_extension_template)
-
-        node: Node = environment.nodes[0]
-        runbook = node.capability.get_extended_runbook(AzureNodeSchema)
-        if re.match(cls._amd_supported_skus, runbook.vm_size):
-            # skip AMD, because no AMD GPU Linux extension.
-            ...
+    def _install_driver_using_platform_feature(self) -> None:
+        supported_versions: Dict[Any, List[str]] = {
+            Redhat: ["7.3", "7.4", "7.5", "7.6", "7.7", "7.8"],
+            Ubuntu: ["16.04", "18.04", "20.04"],
+            CentOs: ["7.3", "7.4", "7.5", "7.6", "7.7", "7.8"],
+        }
+        release = self._node.os.information.release
+        if release not in supported_versions.get(type(self._node.os), []):
+            raise LisaException("GPUExtensionNotSupported")
```

Comment:
Give a more specific error message. The Azure GPU extension doesn't support this distro version. The "NotSupported" here is not clear enough.